yt_dlp/postprocessor/sponsorblock.py

```diff
@@ -10,18 +10,23 @@
 
 
 class SponsorBlockPP(FFmpegPostProcessor):
-
+    # https://wiki.sponsor.ajay.app/w/Types
     EXTRACTORS = {
         'Youtube': 'YouTube',
     }
+    POI_CATEGORIES = {
+        'poi_highlight': 'Highlight',
```

Comment:
Are there gonna be multiple POI categories?

Reply:
There are no plans for any other so far afaik. But having it defined separately makes it easier for us if one is added